conformance_tests/tools/metrics/src/test_metric.cpp

```diff
@@ -2063,29 +2077,51 @@ TEST_F(
           FAIL() << "zeEventQueryStatus() FAILED with " << eventResult;
         }
 
-        std::vector<uint8_t> rawData;
-        uint32_t rawDataSize = 0;
-        lzt::metric_streamer_read_data(metricStreamerHandle, rawDataSize,
-                                       &rawData);
-
-        rawData.resize(rawDataSize);
-        std::vector<zet_typed_value_t> metricValues;
-        std::vector<uint32_t> metricValueSets;
-        lzt::metric_calculate_metric_values_from_raw_data(
-            groupInfo.metricGroupHandle, rawData, metricValues,
-            metricValueSets);
-
-        std::vector<zet_metric_handle_t> metricHandles;
-        lzt::metric_get_metric_handles_from_metric_group(
-            groupInfo.metricGroupHandle, metricHandles);
-        std::vector<zet_metric_properties_t> metricProperties(
-            metricHandles.size());
-        lzt::metric_get_metric_properties_for_metric_group(metricHandles,
-                                                           metricProperties);
-
-        lzt::metric_validate_streamer_marker_data(metricProperties,
-                                                  metricValues, metricValueSets,
-                                                  streamerMarkerValues);
+        const char *value_string =
+            std::getenv("LZT_METRIC_READ_DATA_MAX_DURATION_MS");
+        uint32_t max_duration_in_milliseconds = 10;
```

Comment:
Please change max_duration_in_milliseconds -> max_wait_time_in_milliseconds, if it makes sense